pkg/model/insight.proto

```diff
@@ -18,6 +18,7 @@ package model;
 option go_package = "github.com/pipe-cd/pipecd/pkg/model";
 
 import "validate/validate.proto";
+import "pkg/model/deployment.proto";
 
 enum InsightResultType {
```

Comment:
not really related to this PR, but do we use this? 👀

Reply:
Currently `model.DeploymentStatus` is used.
But maybe we should define own type.

Reply:
Then lets define it when we need it and remove for now 😄